.werft/util/werft.ts

```diff
@@ -115,4 +118,23 @@ export class Werft {
         this.endPhase()
         this.rootSpan.end()
     }
+
+    /**
+     * This allows you to set attributes on all open and future Werft spans.
+     * Any spans in phases that have already been closed won't get the attributes.
+     */
+    public addGlobalAttributes(attributes: SpanAttributes): void {
```

Comment:
We are currently using it to add attributes that we call "global", but the method can be used to add any attribute, right?

```suggestion
    public addAttributes(attributes: SpanAttributes): void {
```